library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt

```diff
@@ -80,6 +81,7 @@ public fun SelectableCalendar(
   monthContainer: @Composable (content: @Composable (PaddingValues) -> Unit) -> Unit = { content ->
     Box { content(PaddingValues()) }
   },
+  swingCallback: (YearMonth) -> Unit,
```

Comment:
In other places we name this gesture `swipe`. Also the postfix `callback` is pretty redundant - we see that it is a callback via the function signature. So I would change it to `onMonthSwipe`.